recipes/byexample/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "byexample" %}
+{% set version = "8.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 44cc2bd00141e3e7f9ccf82eff025385c58ece02cd36da04bea2f0937d2410bb
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  commands:
+    - byexample --version
+
+about:
+  home: http://github.com/byexamples/byexample
+  license: GPL-3.0
+  license_file: LICENSE
```

Comment:
The ``license`` is not being packaged on pypi. Could you please raise an issue about it on this project?
For now, you can package it manually, please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Reply:
Thanks, I reported it upstream and added a LICENSE to the recipe.